guide/14-deep-learning/inference-only-text-models-in-arcgis-learn.ipynb

```diff
@@ -173,24 +173,7 @@
     "\n",
```

Comment:
<p>@anurag89sharma - as this is the first example of using inference only models in the guide, it is preferable to start with simply the default ZeroShotClassifier() constructor. After the simple case has been demonstrated, you can then introduce the option to pick a backbone for the model and show it later. Also, instead of showing a custom model being picked for all inference only models, you can then keep using the default constructor to keep things simple. That way, both the begineer and advanced audience would be served, while demonstrating the common use case (default constructor). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/981/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>